docs/src/inputs/checkbox/checkbox.jsx

```diff
@@ -105,4 +105,4 @@ function DocsCheckbox(props) {
 
 DocsCheckbox.propTypes = propTypes;
 
-export default DocsCheckbox;
+export default DocsCheckbox;
```

Comment:
Please revert this change and bring back my terminating newline :-)

Reply:
Fixed.

Reply:
Yep, I don't see this file diffed anymore.  Resolving.